.gitignore

```diff
@@ -39,6 +39,8 @@ docker-compose.override.yml
 ## Config
 /config.json
 /apps/prairielearn/config.json
+/apps/workspace-host/config.json
+/apps/grader-host/config.json
```

Comment:
These are not directly related to the change, but to test the workspace I had to use a config in workspace-host (`"workspacePullImagesFromDockerHub": false`) and noticed it wasn't ignored as it should be.